pepper-apis/src/main/java/org/broadinstitute/ddp/content/I18nContentRenderer.java

```diff
@@ -249,28 +228,18 @@ private String render(Handle handle, Long contentTemplateId, Long languageCodeId
     }
 
     public String renderToString(String template, Map<String, String> context) {
-        StringResourceRepository repo = (StringResourceRepository) engine
-                .getApplicationAttribute(StringResourceLoader.REPOSITORY_NAME_DEFAULT);
-        repo.putStringResource(TEMPLATE_NAME, template);
-
         VelocityContext ctx = new VelocityContext(context);
-        Template tmpl = engine.getTemplate(TEMPLATE_NAME);
-
         StringWriter writer = new StringWriter();
-        tmpl.merge(ctx, writer);
-
+        engine.evaluate(ctx, writer, TEMPLATE_NAME, template);
```

Comment:
Acccording to velocity's [documentation](https://velocity.apache.org/engine/1.7/developer-guide.html#using-velocity), we don't need to write out to a string repository and read back out the `Template`. `evaluate()` will render it straight from the given string.